src/main/java/driverutils/Driverinterface.java

```diff
@@ -0,0 +1,7 @@
+package driverutils;
+
+import org.openqa.selenium.WebDriver;
+
+public interface Driverinterface {
```

Comment:
Why you don't use CamelCase here in interface name?
I think you can name it like this:
DriverManager (interface)
-> ChromeDriverManager (impl)
-> FirefoxDriverManager (impl)
-> IeDriverManager(impl)
etc.

Reply:
Refactored